src/backend/common/manipulators/manipulator_base.py

```diff
@@ -340,8 +340,7 @@ def _clearCache(cls, models: Iterable[TModel]) -> None:
                 cls._clearCacheDeferred,
                 all_affected_references,
                 _queue="cache-clearing",
-                # this does not exist in Cloud Tasks
-                # _transactional=ndb.in_transaction(),
+                _transactional=ndb.in_transaction(),
```

Comment:
I would be surprised if this works, because the the library probably attempts to reference transactions in the legacy datastore service, not the cloud one we're using